docs/docs/migrate-from/ibm-watson-to-rasa.mdx

```diff
@@ -5,6 +5,54 @@ title: Rasa as open source alternative to IBM Watson - Migration Tips
 description: Open source alternative to IBM Watson for conversational bots and NLP
 ---
 
-There is no support for IBM Watson yet. However, a group of community members is working on a way
-to use <a className="reference external" href="https://developer.ibm.com/tutorials/learn-how-to-export-import-a-watson-assistant-workspace/" target="_blank">exported IBM Watson workspaces</a>
-in Rasa. If you're interested in that, check out our <a className="reference external" href="https://forum.rasa.com/" target="_blank">Community Forum</a>.
+## Step 1 Export your data from Watson
+
+Follow the <a className="reference external" href="https://developer.ibm.com/tutorials/learn-how-to-export-import-a-watson-assistant-workspace/" target="_blank">instructions</a> and export your bot's data.
+
+## Step 2: Create a Rasa Project
+
+To create a Rasa project, run:
+
+```bash
+rasa init
+```
+
+This will create a directory called `data`.
+Remove the files in this directory, and
+move your json file with Watson data into this directory.
+
+```bash
+rm -r data/*
+mv /path/to/file.json data/
```

Comment:
does watson give you a file called `file.json`? If not here's a suggested rename for clarity:
```suggestion
mv /path/to/exported_watson_file.json data/
```